test/unit/assignment_test.rb

```diff
@@ -169,7 +159,12 @@ class AssignmentTest < ActiveSupport::TestCase
       context 'with an assignment with other groupings' do
         setup do
           @target = Assignment.make({:group_min => 1, :group_max => 1})
-          @target.create_groupings_when_students_work_alone
+          3.times do |i|
```

Comment:
Unused block argument - `i`. You can omit the argument if you don't care about it.